taichi/util/short_name.cpp

```diff
@@ -0,0 +1,16 @@
+#include "short_name.h"
+#include "base64.h"
+
+const std::string alphatable = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"; // e.g. `if` is not a good var name, but `If` does.
```

Comment:
As a general rule of thumb in C++, please wrap such variables that are not exposed with an anonymous namespace. See https://stackoverflow.com/a/357427/12003165